config.py

```diff
@@ -0,0 +1,17 @@
+# add configuration
+# https://docs.pydantic.dev/latest/concepts/pydantic_settings/#usage
+import logging
+import os
+import certifi
+
+
+def configure():
+    print("Configuring.")
+    os.environ["REQUESTS_CA_BUNDLE"] = certifi.where()
+    os.environ['CURL_CA_BUNDLE'] = certifi.where()
+    # os.environ['REQUESTS_CA_BUNDLE'] = r"/etc/ssl/cert.pem"
+    # os.environ['REQUESTS_CA_BUNDLE'] = r"/etc/ssl/openssl.cnf"
```

Comment:
Commented out code either has to have a good explanation next to it or should not be there.

Reply:
Fixed